libs/eavmlib/src/port.erl

```diff
@@ -57,19 +57,18 @@ call(Port, Message) ->
 %%-----------------------------------------------------------------------------
 %% @param   Pid Pid to which to send messages
 %% @param   Message the message to send
-%% @param   TimeoutMs the timeout value in milliseconds
+%% @param   Timeout the timeout value in milliseconds
 %% @returns term() | {error, Reason}.
 %% @doc     Send a message to a given port driver pid with a timeout.
 %%
 %% This function is used to send a message to an open port divers pid and will return
 %% a term or `{error, Reason}', or`{error, timeout}' if the TimeoutMs is reached first.
 %% @end
 %%-----------------------------------------------------------------------------
--spec call(pid(), Message :: term(), TimeoutMs :: non_neg_integer()) -> term() | {error, timeout}.
-call(Port, Message, TimeoutMs) ->
-    Ref = erlang:make_ref(),
+-spec call(pid(), Message :: term(), Timeout :: timeout()) -> term() | {error, timeout}.
+call(Port, Message, Timeout) ->
     MonitorRef = monitor(port, Port),
-    Port ! {self(), Ref, Message},
+    Port ! {self(), MonitorRef, Message},
```

Comment:
Everything looks clear and done accordingly the commit message, however this change is not clear to me.
Is it intentional or did it sneak in from other changes?

Reply:
It was intentional, I updated the commit message accordingly.